docs/05-storage-providers.md

```diff
@@ -412,158 +436,215 @@ Charges and functionality vary with bucket storage class. Known storage classes:
 You can use this backend without this permission by using an AuthID generated for the `SharePoint v2` backend and passing the `--group-id` instead of `--group-email`. You can lookup the group ID by using the [Microsoft Graph Explorer](https://developer.microsoft.com/en-us/graph/graph-explorer).
 *****
 
-This backend can store backups in the document library associated with a Microsoft Office 365 Group, using the following URL format:
+This backend can store backups in the document library associated with a Microsoft Office 365 Group.
+
+URL Format:
 
 `msgroup://folder/subfolder`
 
 Options:
 
-* `--authid (Password)`  
+* `--authid (Password)`
 The authorization token retrieved from [https://duplicati-oauth-handler.appspot.com?type=msgroup](https://duplicati-oauth-handler.appspot.com?type=msgroup)
-* `--group-email (String)`  
-Email address of the group to store data in, for example: `test@example.com`.  
-Either this parameter or `--group-id` is required.  
+* `--group-email (String)`
+Email address of the group to store data in, for example: `test@example.com`.
+Either this parameter or `--group-id` is required.
 When this parameter is specified, the group's ID is looked up automatically.
-* `--group-id (String)`  
-ID of the group to store data in.  
-Either this parameter or `--group-email` is required.  
+* `--group-id (String)`
+ID of the group to store data in.
+Either this parameter or `--group-email` is required.
 This ID can be looked up by querying in the [Microsoft Graph Explorer](https://developer.microsoft.com/en-us/graph/graph-explorer).
-* `--fragment-size (Integer)`  
-Size of individual fragments which are uploaded separately for large files.  
-It is recommended to be between `5-10 MiB` (though a smaller value may work better on a slower or less reliable connection), and to be a multiple of `320 KiB`.  
+* `--fragment-size (Integer)`
+Size of individual fragments which are uploaded separately for large files.
+It is recommended to be between `5-10 MiB` (though a smaller value may work better on a slower or less reliable connection), and to be a multiple of `320 KiB`.
 Default value: `10 MiB`
-* `--fragment-retry-count (Integer)`  
-Number of retry attempts made for each fragment before failing the overall file upload.  
+* `--fragment-retry-count (Integer)`
+Number of retry attempts made for each fragment before failing the overall file upload.
 Default value: `5`
-* `--fragment-retry-delay (Integer)`  
-Amount of time (in milliseconds) to wait between failures when uploading fragments.  
+* `--fragment-retry-delay (Integer)`
+Amount of time (in milliseconds) to wait between failures when uploading fragments.
 Default value: `1000`
 
-## Microsoft OneDrive v2 (Microsoft Graph API)

```

Comment:
I just moved the OneDrive and Sharepoint documentation so that the v1 and v2 versions are next to each other.  No deletes here.